photon-ml/src/integTest/scala/com/linkedin/photon/ml/BootstrapTrainingIntegTest.scala

```diff
@@ -87,67 +89,4 @@ class BootstrapTrainingIntegTest extends SparkTestUtils {
       }
     })
   }
-
-  /**
-   * "Real" integration test where we hook in all the aggregation operations and sanity check their output
-   */
-  @Test
-  def checkBootstrapHappyPathRealAggregates(): Unit = sparkTest("checkBootstrapHappyPathRealAggregates") {
-    // Return a different model each time fitFunction is called
-    var count: Int = -BootstrapTrainingTest.HALF_NUM_SAMPLES
-    val fitFunction = (x: RDD[LabeledPoint], y: Map[Double, LinearRegressionModel]) => {
-      val value = count / BootstrapTrainingTest.HALF_NUM_SAMPLES.toDouble
-      count += 1
-      val fn = regressionModelFitFunction(value, lambdas)
-      fn(x, y)
-    }
-
-
-    val confidenceIntervalsKey = "confidenceIntervalEstimate"
-    val metricsIntervalsKey = "metricsIntervalEstimate"
-    val aggregations: Map[String, Seq[(LinearRegressionModel, Map[String, Double])] => Any] = Map(
-      confidenceIntervalsKey -> BootstrapTraining.aggregateCoefficientConfidenceIntervals,
-      metricsIntervalsKey -> BootstrapTraining.aggregateMetricsConfidenceIntervals
-    )
-
-    val validateConfidenceIntervals: Any => Unit = x => {
-      x match {
-        case (coeff: Array[CoefficientSummary], intercept: Option[CoefficientSummary]) =>
-          coeff.foreach(c => {
-            BootstrapTrainingTest.checkCoefficientSummary(c)
-          })
-          intercept match {
-            case Some(_) => fail("Intercept should not have been computed")
-            case None =>
-          }
-        case _ =>
-          fail(s"Aggregate for $confidenceIntervalsKey is of unexpected type")
-      }
-    }: Unit
-
-    val validateMetricsIntervals: Any => Unit = x => {
-      x match {
-        case m: Map[String, CoefficientSummary] =>
-        case _ =>
-          fail(s"Aggregate for $metricsIntervalsKey is of unexpected type")
-      }
-    }: Unit
-
-    val aggregationValidators: Map[String, Any => Unit] = Map(
-      confidenceIntervalsKey -> validateConfidenceIntervals,
-      metricsIntervalsKey -> validateMetricsIntervals
-    )
-
-    val data: RDD[LabeledPoint] = sc.parallelize(drawSampleFromNumericallyBenignDenseFeaturesForLinearRegressionLocal(seed.toInt, numSamples, BootstrapTrainingTest.NUM_DIMENSIONS).toSeq).map(x => new LabeledPoint(x._1, x._2)).coalesce(4)
-
-    val aggregates: Map[Double, Map[String, Any]] = BootstrapTraining.bootstrap[LinearRegressionModel](
-      BootstrapTrainingTest.NUM_SAMPLES,
-      samplePct,
-      Map[Double, LinearRegressionModel](),
-      fitFunction,
-      aggregations,
-      data)
-  }
 }
```

Comment:
We know that the IntegTest is necessary here for BootstrapTraining class, given the requirement of spark context. But I am asking because I didn't see ```aggregates: Map[Double, Map[String, Any]]``` used anywhere in the test to check whether we got expected results, and in the comment (line 91-93) it's being said that "and  sanity check their output", but I didn't see the sanity check. For example, from line 67-89 from the previous tests we know how the testing results are checked. 

As a result, what we see are a few vals (e.g., line 136-139) hanging there being defined but not used. And the only chance (please correct me if I am wrong) this test would fail is when those sanity checks found in line 156-162 of BootstrapTraining.scala fail, or because of some Spark internal runtime failures.

However, I think you made a good point here. Let me add these lines of code back and let's provide a few more sanity checking for the tests in a separate PR in order to make it even more useful.